vm/templates/script/Cargo.toml

```
@@ -0,0 +1,21 @@
+[workspace]
+[package]
+version = "0.1.0"
+name = "wallet-script"
+edition = "2021"
+default-run = "execute"
+
+[[bin]]
+name = "execute"
+path = "src/bin/execute.rs"
+
+[dependencies]
+athena-interface = { git = "https://github.com/athenavm/athena.git", branch = "main" }
+athena-sdk = { git = "https://github.com/athenavm/athena.git", branch = "main" }
+serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+clap = { version = "4.0", features = ["derive", "env"] }
+wallet-common = { path = "../common" }
```

NIT: I think these aren't used, are they? But anyway both program and script need to be rewritten so meh :shrug: 
```suggestion
clap = { version = "4.0", features = ["derive", "env"] }
```